extensions/amp-analytics/0.1/test/test-amp-analytics.js

```diff
@@ -32,6 +32,7 @@ import {
 } from '../../../../src/service/url-replacements-impl';
 import * as sinon from 'sinon';
 
+/* global require: false */
```

Comment:
yeah i can do that, but will probably do it after everything else is cleaned up. hard to verify with all the lint noise